msteams-platform/concepts/design/plan-responsive-tabs-for-teams-mobile.md

```diff
@@ -59,7 +59,7 @@ You can show the following information:
 * Data visuals, maps, and infographics.
 * Charts, graphs, and tables.
 
-:::image type="content" source="../../assets/images/app-fundamentals/dashboarding-and-data-visualization-apps-consumption.png" alt-text="Show the data in the form of visualization.":::
+:::image type="content" source="../../assets/images/app-fundamentals/dashboarding-and-data-visualization-apps-consumption.png" alt-text="The screenshot is an example that shows the data in the form of visualization.":::
```

Comment:
Not a scrnshot. Not an example.
Shows data as visualization may not be a good alt-text to describe the image.

CC @v-habibaumme

Reply:
changes screenshot to image